src/components/chip/__tests__/__snapshots__/chip.test.tsx.snap

```diff
@@ -1,33 +1,35 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`Chip component should match a snapshot 1`] = `
+exports[`Chip component > should match a snapshot 1`] = `
 {
   "asFragment": [Function],
   "baseElement": <body>
```

Comment:
**note:** I have not swept through all the existing snapshot tests to update them to use `asFragment` instead of capturing the whole `render` result. I will deal with that via #157 once this PR is merged in.